tests/sentry/integrations/gitlab/test_integration.py

```diff
@@ -108,9 +108,11 @@ def assert_setup_flow(self, user_id="user_id_1"):
         assert req_params["client_id"] == ["client_id"]
         assert req_params["client_secret"] == ["client_secret"]
 
-        assert resp.status_code == 200
-
-        self.assertDialogSuccess(resp)
+        assert resp.status_code == 302
+        assert (
+            resp["Location"]
+            == f"http://testserver/settings/{self.organization.slug}/integrations/gitlab/"
+        )
```

Comment:
I have to do this instead of `resp.url` because it was failing the mypy monkey patch, and this is a recommended work around https://github.com/typeddjango/django-stubs/issues/971#issuecomment-1244480175